.github/meta_issue_template/release-1.1.1-plugin-issues.md

```diff
@@ -0,0 +1,35 @@
+Coming from [opensearch-build#870](https://github.com/opensearch-project/opensearch-build/issues/870), release version 1.1.1. Please follow the following checklist.
+### You can find all the date in above issue 
+### (We only make changes to OpenSearch-Dashboards to 1.1.1, therefore, OpenSearch stays 1.1.0.)
+
+### Preparation
+
+- [ ] Assign this issue to a release owner.
+- [ ] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1).
+- [ ] Create, update, triage and label all features and issues targeted for this release with v1.1.1.
+
+### CI/CD
+
+- [ ] Increment version on main to `1.1.1.0`.
+- [ ] Ensure working and passing CI.
+- [ ] Re(add) this repo to the [manifest](https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1).
```

Comment:
Since we're only making a change to Dashboards, reference the actual dashboards manifest, which I think needs to be created. You can do it as part of this PR, mimic how opensearch-1.1.1.yml looks like. Since this is a patch and the newly created .yml will contain distribution references, this line should say "Change the build from distribution to build from source in the 1.1.1 manifest."

Reply:
I copied from 1.2.0 dashboards manifest so no need for them to change from dist to source, since there is no dist for 1.1.0 dashboards.